core/server/master/src/main/java/alluxio/master/file/FileSystemMaster.java

```diff
@@ -210,24 +211,26 @@
    * (A) cannot call (D)
    *
    * (B) private (or package private) methods that journal:
-   * These methods perform the work from the public apis, and also asynchronously write to the
-   * journal (for write operations). The names of these methods are suffixed with "AndJournal".
+   * These methods perform the work from the public apis, like checking and error handling, and also
+   * asynchronously append entries to the journal. The names of these methods are suffixed with
+   * "AndJournal".
    * (B) cannot call (A)
    * (B) can call (B)
    * (B) can call (C)
    * (B) cannot call (D)
    *
    * (C) private (or package private) internal methods:
-   * These methods perform the rest of the work, and do not do any journaling. The names of these
-   * methods are suffixed by "Internal".
+   * These methods perform the rest of the work, and may append to the journal. The names of these
```

Comment:
That is a good question. I do think there is a difference between (B) and (C), because all the `entry` methods will call (C), and not (B). I think (B) can be inlined with (A). That type of refactor is independent and can be done at a later time.